### ISSUE TITLE
no results showing up on sniper pro 8

### ISSUE THREAD

**Issue by digitizeddude** [1N3/Sn1per#336]
Sniper pro 8 is loading but there are no results when i run any of the scans inside sniper pro 8. There is loot but no results on the web interface. This is consistent regardless of whether i'm running the command inside the terminal or the web interface. I rebooted the kali box and i tried to run the command again from the sniper web interface and not i get the following error:

This site can’t be reachedThe webpage at https://127.0.0.1:1337/pro/addons/server-scan.php might be temporarily down or it may have moved permanently to a new web address.
ERR_CONTENT_DECODING_FAILED

I checked to see if the server-scan.php file is inside the folder /usr/share/sniper/pro/addons and the php file is there.

Not really sure what is going on and what broke but the last major item that i did was add the pip module for zap and  sniper pro started acting this way.

The environment is:
kali 2020.4
sniper 9
sniper pro 8

Thanks.

**Comment by digitizeddude** [comment]
So far, i've added more ram and processing power because it seemed like it was a resource issue and i'm still running in to the same problems. I've also run an update && upgrade and no change.

**Comment by digitizeddude** [comment]
![resultspage-is-messed-up](https://user-images.githubusercontent.com/74317324/105942202-c54e4280-602c-11eb-8514-248a90b8a119.png)

I didn't realize this until i finally got a chance to use Sniper Pro but the reports from Sniper Pro are causing crashes for the browser (I've used Chromium and Firefox) and when i finally do get the sub menu for the reports its hit or miss if i get the report to display. Not sure what is going on. I've deployed sniper before but ive had nothing but issues on a new install on kali 2020.4. I'd try with kail 2020.3 but eventually 2020.3 will just become obsolete so i'd prefer to stick with a solution on 2020.4.
